## ETH-Z Dataset Geometry
{
  ## Initial State
x0: [0, 0, -0.2,    # POS
     0, 0, 0,         # VEL
     1, 0, 0, 0,      # ATT
     -0.33, 0.228, 0.004, # B_A
     0.09, 0.01, 0.004,  # B_G
     1.0],           # MU

image_size: [752, 480],
cam_center: [367.215, 248.375],
focal_len: [458.654, 457.296],
distortion: [-0.28340811, 0.07395907, 0.00019359, 1.76187114e-05],
invert_image: false,

q_b_IMU: [1, 0, 0, 0],
q_I_truth: [1, 0, 0, 0],

q_b_c: [0.567984, -0.396703, -0.419721, -0.586392],
p_b_c: [-0.017648, -0.064100, -0.018095],


feature_mask: /home/superjax/dev_ws/src/vi_ekf/params/firefly.png,  
}